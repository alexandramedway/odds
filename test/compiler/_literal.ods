state 42
state 42.1
state "forty-two"

state print(42)
state print(42.1)
state print("forty-two")
