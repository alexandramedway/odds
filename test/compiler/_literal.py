42
42.1
"forty-two"
None
print(42)
print(42.1)
print("forty-two")
print(None)
