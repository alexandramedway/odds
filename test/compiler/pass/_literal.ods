do 42
do 42.1
do "forty-two"
do void

do print(42)
do print(42.1)
do print("forty-two")
do print(void)
