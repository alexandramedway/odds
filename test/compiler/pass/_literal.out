42
42.1
forty-two
